src/main/webapp/app/overview/student-questions/student-question-row.component.html

```diff
@@ -63,53 +73,102 @@ <h6 class="text-black-50 question-subtitle">{{ studentQuestion.author.name }} -
                 </div>
             </div>
         </div>
+        <!-- approved answers -->
         <div class="row mb-1 answer-approved" *ngFor="let approvedQuestionAnswer of approvedQuestionAnswers">
-            <div class="col" [ngbTooltip]="'artemisApp.courseOverview.exerciseDetails.faq.approvedLabel' | translate">
+            <div class="col">
                 <div *ngIf="isAnswerMode && selectedQuestionAnswer && approvedQuestionAnswer && selectedQuestionAnswer.id === approvedQuestionAnswer.id">
                     <div class="col-12">
```

Comment:
A bootstrap `.col` should always be a child of a `.row`.

Reply:
fixed with https://github.com/ls1intum/Artemis/pull/1325/commits/cb8b3bbfc0d8d43792ef8f1033ee0fbfcb125008